c2corg_api/views/feed.py

```diff
@@ -49,23 +52,53 @@ def get(self):
             http://www.servicedenuages.fr/pagination-continuation-token (fr)
 
         """
-        lang = self.request.validated.get('lang')
-        token_id = self.request.validated.get('token_id')
-        token_time = self.request.validated.get('token_time')
-        limit = self.request.validated.get('limit')
-        limit = min(
-            DEFAULT_PAGE_LIMIT if limit is None else limit,
-            MAX_PAGE_LIMIT)
-
-        changes = get_changes_of_public_feed(token_id, token_time, limit)
+        lang, token_id, token_time, limit = get_params(self.request)
+        changes = get_changes_of_feed(token_id, token_time, limit)
         return load_feed(changes, lang)
 
 
-def get_changes_of_public_feed(token_id, token_time, limit):
+@resource(path='/personal-feed', cors_policy=cors_policy)
+class PersonalFeedRest(object):
+
+    def __init__(self, request):
+        self.request = request
+
+    @restricted_view(validators=[
+        validate_preferred_lang_param, validate_token_pagination])
+    def get(self):
+        """Get the personal homepage feed for the authenticated user.
```

Comment:
I wanted to have different URLs `/feed` and `/personal-feed` mainly for caching reasons. Because the public feed can be cached by Varnish (for a short period, eg 30 seconds).